test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -497,6 +497,38 @@ def forward(self, x):
         x = torch.randn(2, 3, 10, 50, 100, requires_grad=True)
         self.run_test(model, (x,), rtol=1e-3, atol=1e-6)
 
+    def test_fuse_conv_in_block(self):
+        class Fuse(torch.nn.Module):
+            def __init__(self):
+                super(Fuse, self).__init__()
+                self.conv = torch.nn.Conv1d(
+                    in_channels=5,
+                    out_channels=5,
+                    kernel_size=3,
+                    stride=1,
+                    padding=2,
+                    dilation=1
+                )
+                self.bn = torch.nn.BatchNorm1d(5)
+
+            def forward(self, x):
+                results_available = True
+
+                if x.sum() > -1:
+                    results_available = False
+
+                if results_available:
+                    x = self.conv(x)
+                    x = self.bn(x)
+
+                return x
+
+        model = Fuse()
+        x = torch.randn(2, 5, 9, requires_grad=True)
+        self.run_test(torch.jit.script(model), (x,),
```

Comment:
I guess run_test will run the test twice unnecessarily:
https://github.com/pytorch/pytorch/blob/onnx_ms_1/test/onnx/test_pytorch_onnx_onnxruntime.py#L317

Not sure if this is the cleanest way to do it, but perhaps we could change the code to 

`if self.is_script_test_enabled and not isinstance(model, ScriptModule):`

I'd actually like to rework how we run the script tests in a larger way because today it's not clear from the error message nor test name whether the failure is in scripting mode or not. LMK if you have any ideas to improve that.